### PR TITLE
96051 create modal

### PR DESCRIPTION
**Pull request by numbap**
### Description

List of proposed changes:

- Created CountDown and SignedOut modal.  
- Created temporary modaldemo page to trigger modals. 
-

### What to test for/How to test

Unit tests create a snapshot of current modal state.

### CODE REVIEW

**Review by shewood — COMMENTED** [review]
Is there a way to test this without a snapshot test? They are hard to maintain.

File: components/sessionModals/CountDown.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        <div className="flex-none flex items-top justify-center pr-3">
+          <FontAwesomeIcon
+            icon={icon['triangle-exclamation']}
+            size="lg"
+            color="orange"
+          />
+        </div>
+        <div className="flex-auto">
+          <p className="mr-6">
+            You have been active for a while so we will sign you out to keep
```
Comment (nibivi77) [inline components/sessionModals/CountDown.js:72]: I assume we mean "inactive'? and not "active"

Do we know if we will get this from AEM?

**Review by nibivi77 — COMMENTED** [review]
`CountDown.js` and `SignedOut.js` are very similar to `ExitBetaModal.js`.

Could we find a way to reduce code duplication and combine them in a common Modal component? 

Just thoughts.. we can discuss this !

File: components/sessionModals/CountDown.js (hunk 1 above)
Comment (numbap) [reply components/sessionModals/CountDown.js]: Yes. The content will come from AEM.

**Review by nibivi77 — APPROVED** [review]

**Pull request merged**